### PR TITLE
Fixing and addittions to Onboarding-CLI doc

### PR DESCRIPTION
**Pull request by Ryushka**
## Summary

Few edits that remove deleted command, fixing typos in commands, adding just introduced asset command, a little addition to explanation of miners command

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: docs/onboarding/cli.md [hunk 1]
```diff
@@ -253,6 +256,14 @@ ironfish faucet
 ```
 
 ### Chain
+
+#### chain:asset
+Get the asset info by asset identifier
+
+```sh
+ironfish chain:asset <identifier of an existing asset>
```
Comment (ygao76) [inline docs/onboarding/cli.md:264]: can you have dash to keep the consistency of the doc, so <identifier-of-an-existing-asset>

File: docs/onboarding/cli.md [hunk 2]
```diff
@@ -101,34 +104,34 @@ config:get enableMetrics
...
+Setting the number of CPU threads to utilize for mining (-t) and setting other than your default account address for block rewards (-a)
 
 ```sh
-ironfish miners:start -p <ip-address-of-pool>
+ironfish miners:start -t <number of threads to use> -a <your address to receive rewards>
 ```
 
-#### miners:pool:start
-Starts a mining pool with the name set in the `poolName` configuration option.
+Join a mining pool with the your default account public address.
```
Comment (ygao76) [inline docs/onboarding/cli.md:116]: the original words are correct. This command starts your own mining pool.

File: docs/onboarding/cli.md [hunk 3]
```diff
@@ -101,34 +104,34 @@ config:get enableMetrics
 Starts a miner and subscribes to new blocks for the node. The node has to be synced with the network for the miner to start mining.
 
 ```sh
-ironfish miners:start
+ironfish miners:start  
 ```
 
-Join a mining pool.
```
Comment (ygao76) [inline docs/onboarding/cli.md:107]: Can you keep the Join a mining pool? so 
Join a mining pool with the number of CPU threads to utilize for mining (-t) and setting other than your default account address for block rewards (-a)

`ironfish miners:start` is joining a mining pool. `ironfish miners:pools` is starting your own mining pool.

**Review by ygao76 — APPROVED** [review]

**Comment by Ryushka** [comment]
I'm sorry but changes you've made are wrong... Without flag <-p> you can't join a pool...
ironfish miners:start -t <number-of-threads-to-use> -a <your-address-to-receive-rewards> 
Is a command to launch solo miner with specified number of threads and other than default wallet address...

Join a mining pool with setting the number of CPU threads to utilize for mining <-t> and setting other than your default account address for block rewards <-a>.
ironfish miners:start -t <number-of-threads-to-use> -a <your-address-to-receive-rewards>

**Comment by dguenther** [comment]
@ygao76 Could you make a follow-up PR to fix that last commit? I think @Ryushka is right that the description of the command that was changed ("Join a mining pool ...") doesn't actually join a mining pool.

**Comment by ygao76** [comment]
Okay!

**Comment by NullSoldier** [comment]
@Ryushka What graffiti do you want this credited to?

**Comment by Ryushka** [comment]
> 
PaRUSaI

**Pull request merged**